.github/workflows/macos-microshift.yml

```diff
@@ -35,6 +35,18 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
+      - name: check if OSP_PULL_SECRET exists
+        shell: bash
+        if: ${{ env.PULL_SECRET == '' }}
+        run: 'echo the secret \"OCP_PULL_SECRET\" has not been made; echo please go to \"settings \> secrets \> actions\" to create it'
+        env:
+          PULL_SECRET: ${{ secrets.OSP_PULL_SECRET }}
```

Comment:
Not clear why this is named `OSP_PULL_SECRET`, and why the error message mentions `OCP_PULL_SECRET`

Reply:
OSP => openshift platform will change error message.

Reply:
RHOSP is OpenStack, OpenShift is RHOCP.

Reply:
In my brain somehow OSP stuck that time what I really meant OCP => Openshift container platform :(